core/api_logs.go

```diff
@@ -0,0 +1,54 @@
+package core
+
+import (
+	logging "gx/ipfs/QmZChCsSt8DctjceaL56Eibc29CVQq4dGKRXC5JRZ6Ppae/go-log"
+	logger "gx/ipfs/QmcaSwFc5RBg8yCq54QURwEU4nwjfCpjbpmaAm4VbdGLKv/go-logging"
+	"net/http"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+)
+
+func (a *api) logsCall(g *gin.Context) {
+
+	subsystem := g.Param("subsystem")
+	var subsystems []string
+	if subsystem == "" {
+		subsystems = logging.GetSubsystems()
+	} else {
+		subsystems = []string{subsystem}
+	}
+
+	opts, err := a.readOpts(g)
+	if err != nil {
+		a.abort500(g, err)
+		return
+	}
+	level := strings.ToUpper(opts["level"])
+	all := opts["all"] == "true"
+
+	var result []map[string]string
+	for _, system := range subsystems {
+		if strings.HasPrefix(system, "tex") || all {
```

Comment:
would be nice to be able to drill into IPFS systems as well... as in, info _only_ from pubsub or something. does this restrict that?

Reply:
nope, you can still do something like:

```
textile logs -s pusub -l info --all
```

to change only `"pubsub"` to `"info"` (note the `--all` flag). Reading that now though, maybe `all` isn't the right name for it?

Reply:
this is even more straight forward with latest push